src/Mvc/Mvc.Formatters.Json/src/JsonFormatterOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.AspNetCore.Mvc.Formatters
+{
+    /// <summary>
+    /// Provides options to configure formatters using <c>System.Text.Json</c>.
+    /// </summary>
+    public class JsonFormatterOptions
+    {
+        /// <summary>
+        /// Gets the <see cref="JsonSerializerOptions"/>.
+        /// </summary>
+        public JsonSerializerOptions SerializerOptions { get; } = new JsonSerializerOptions
```

Comment:
implement real property and make this not possible to set, but not to null? I can see someone wanting to replace this object completely.

Reply:
Per https://github.com/aspnet/Mvc/issues/6816#issuecomment-329221624, we made a conscious choice not to allow this with the Newtonsoft.Json SerializerSettings. The same arguments apply here.